xkcdpass/xkcd_password.py

```diff
@@ -191,9 +190,9 @@ def find_acrostic(acrostic, worddict):
     words = []
 
     for letter in acrostic:
-        try:
+        if letter in worddict:                        #  better than try catch blocks
```

Comment:
Again, python's try...except is best practice here: we are raising a crashing error in the infrequent worst case rather than testing every loop